apps/redi-connect/src/components/organisms/Footer.tsx

```diff
@@ -34,15 +35,24 @@ const RediFooter = () => {
               >
                 <RediSchool />
               </Element>
-              <Element
-                renderAs="a"
-                href="https://www2.deloitte.com/"
-                target="_blank"
-                className="footer__logo"
-              >
-                <Deloitte />
-              </Element>
+              <Columns.Column mobile={{ size: 12 }}>
+                <Element
+                  alt="Supported by Deloitte"
+                  style={{ maxWidth: '6rem' }}
```

Comment:
Can we move this CSS rule into the `Footer.scss` file? Same goes for the various places we have inline CSS.